packages/odyssey-react-mui/src/Dialog.tsx

```diff
@@ -40,14 +40,21 @@ export type DialogProps = {
    * An optional Button object to be situated in the Dialog footer. Should almost always be of variant `primary`.
    */
   primaryCallToActionComponent?: ReactElement<typeof Button>;
+  /** @deprecated Will be removed in a future Odyssey version. Use `primaryCallToActionComponent` instead. */
+  callToActionFirstComponent?: ReactElement<typeof Button>;
   /**
    * An optional Button object to be situated in the Dialog footer, alongside the `callToActionPrimaryComponent`.
    */
   secondaryCallToActionComponent?: ReactElement<typeof Button>;
+  /** @deprecated Will be removed in a future Odyssey version. Use `secondaryCallToActionComponent` instead. */
+  callToActionSecondComponent?: ReactElement<typeof Button>;
+
   /**
    * An optional Button object to be situated in the Dialog footer, alongside the other two `callToAction` components.
    */
   tertiaryCallToActionComponent?: ReactElement<typeof Button>;
+  /** @deprecated Will be removed in a future Odyssey version. Use `tertiaryCallToActionComponent` instead. */
+  callToActionLastComponent?: ReactElement<typeof Button>;
```

Comment:
@benlister-okta I think we need to tighten up the types here a bit so someone can't use both `tertiaryCallToActionComponent` AND `callToActionLastComponent`. Applies for all 3 props